testsuite/testsuite-jsp/pom.xml

```diff
@@ -49,6 +64,12 @@
             <version>${project.version}</version>
             <executions>
               <execution>
+                <id>package</id>
```

Comment:
No, `package` is not executed by default. All the tests currently run based off `wildfly-swarm:start/stop`. 

I'll move the test to the `uberjar` profile